lib/core/Core.ts

```diff
@@ -112,8 +110,7 @@ export default class Core {
   public async handleGetVersionRequest (): Promise<ResponseModel> {
     const responses = [
       this.serviceInfo.getServiceVersion(),
-      await this.blockchain.getServiceVersion(),
-      await this.cas.getServiceVersion()
+      await this.blockchain.getServiceVersion()
```

Comment:
Dicussion: Do we care about cas version at all? For example, ICas can have a getVersion() function and our implementation will just return the go ipfs version. Just a thought. Don't have to do it in the pr even if we want it.

Reply:
This was written specifically for our microservices, but I can see an argument for surfacing go-ipfs version here maybe? Though that opens up the question of if we should do the same for Bitcoin Core... happy to discuss this for a separate PR.

Reply:
Yea maybe the bitcoin, ipfs can call the dependent services and return their versions as well. Not a bad idea.